community-operators/ham-deploy/0.1.0/ham-deploy.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,147 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: ham-deploy.v0.1.0
+  namespace: placeholder
+  annotations:
+    alm-examples: |-
+      [
+      {
+        "apiVersion": "tools.hybridapp.io/v1alpha1",
+        "kind": "Operator",
+        "metadata": {
+          "name": "simple"
+        },
+        "spec": {
+          "core": {
+            "deployable": {
+              "enabled": false
+            }
+          },
+          "tools": {
+            "assembler": {
+              "enabled": true
+            },
+            "discoverer": {
+              "enabled": false,
+              "clustername": "toronto",
+              "clusternamespace": "toronto-ns"
+            }
+          }
+        }
+      }
+      ]
+    categories: Integration & Delivery, OpenShift Optional
+    certified: 'false'
+    createdAt: 2020-05-12 00:00:00
+    description: Deploys Hybrid Application Model
+    containerImage: quay.io/hybridappio/ham-deploy@sha256:2fc36d30c4b10cc2834d00a703581f579dc2ecd28cb5e3e67be35d773b420b02
+    support: hybridapp-io@googlegroups.com
+    capabilities: Basic Install
+    repository: https://github.com/hybridapp-io/
+spec:
+  displayName: Hybrid Application Model Operator
+  description: |
+    ## About Multicloud Manager Model
+
+    Multicloud Manager Model introduces infrasctructure management on top of the Multicluster Subscription model. See the links for more details.  
+    The model consists of an Appliction Assembler, Hybrid Deployable, and a Resource Discoverer.
+
+    ## About this Operator
+
+    This operator installs one or more components for the hub cluster (Application Assembler, Deployable are required, Resource Discoverer is optional).  Managed clusters should only install the Resource Discoverer.
+
+    ## Prerequisites for enabling this Operator
+
+    Requires Multicluster Subscription Operator 1.0+
+
+  maturity: alpha
+  version: 0.1.0
+  replaces: ''
```

Comment:
If this is first version of operator, one should remove line with `replaces: ''`. Please remove it for this pull request and use it only when you will update new version of operator. Thanks